src/inverseRobot.js

```diff
@@ -7,7 +7,46 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const outRobot = { ...robot };
+  const newRobot = {};
+  const keysArr = [];
+  const valuesArr = [];
```

Comment:
Try to make up better names for variables(but you don't need so many, try to refactor this)